src/ui/composer.ui

```diff
@@ -124,11 +133,7 @@
           </sizepolicy>
          </property>
          <property name="toolTip">
-          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
-&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
-p, li { white-space: pre-wrap; }
-&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
-&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Save (&lt;span style=&quot; color:#565656;&quot;&gt;Shift+Tab&lt;/span&gt;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+          <string>&lt;p&gt;Save (&lt;span style=&quot; color:#565656;&quot;&gt;Shift+Tab&lt;/span&gt;)&lt;/p&gt;</string>
```

Comment:
From my experience I think you mean/need the `<html><head/><body>`...`</body></html>` wrapping as well so that it becomes (I think):
```
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save (&lt;span style=&quot; color:#565656;&quot;&gt;Shift+Tab&lt;/span&gt;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```
The `<Shift>-<Tab>` short-cut worries me a little because for the main application doesn't that switches to the last (previously) active profile when multi-playing - or is this *composer*(?) a separate free-floating dialogue?  Also - why is the short-cut in a dark-grey colour - that seems rather to specific and may interfere with more global styling options - indeed, why bother colouring just that bit at all?

Reply:
It didn't seem necessary to wrap it in that in my testing. Is it needed for you?

The colours make it look nice, hasn't been an issue. This is just about making it easier for translators.